corehq/apps/registration/forms.py

```diff
@@ -536,7 +536,8 @@ def __init__(self, data=None, excluded_emails=None, is_add_user=None,
                 'profile' if ('profile' in self.fields and len(self.fields['profile'].choices) > 0) else None,
             )
         ]
-        if should_show_location:
+        loc_flag_enabled = toggles.LOCATION_FIELD_USER_PROVISIONING.enabled(domain, toggles.NAMESPACE_DOMAIN)
+        if should_show_location and loc_flag_enabled:
```

Comment:
Wait, doesn't this mean that projects without the flag won't be able to edit user locations at all?

Reply:
> This just removes the location section from the Web User invite form.

That's what I meant to say - I didn't think this was the goal, just to feature flag the very recent changes to the widget itself (changing the trunctation, using the display name instead of path, removing already-selected locations).  Locations have been available in invitations [since May](https://github.com/dimagi/commcare-hq/pull/34466), so removing those from view seems like it might be disruptive.  I'll double check though.

Reply:
I see. Thanks for checking. I see Ben confirmed also,